tests/platform_tests/mellanox/util.py

```diff
@@ -215,7 +215,8 @@ def check_sfp_eeprom_info(duthost, sfp_eeprom_info, is_support_dom, show_eeprom_
                 ["Application Advertisement", "ChannelThresholdValues", "ModuleThresholdValues"])
             expected_keys = expected_keys - excluded_keys
 
-    if "Identifier" in sfp_eeprom_info and sfp_eeprom_info["Identifier"] == "SFP/SFP+/SFP28":
+    sfp_type_for_excluded_monitor_threshold_key = ["SFP/SFP+/SFP28", "OSFP 8X Pluggable Transceiver"]
```

Comment:
Since the output doesn't include the four keys, I skip checking them.
I have checked it with @keboliu. We think for passive cable it should not include the four keys, otherwise, it should include.
So, we can change the above code to skip check these keys if the cable is a passive cable. Is it ok?

Reply:
@JibinBao yes we should skip checking for channel monitor values for DAC cables

Reply:
thanks. Will update it

Reply:
@prgeor
We have one issue currently: https://github.com/sonic-net/sonic-buildimage/issues/14602
Maybe after the issue is fixed or the design is figured out, we can change it accordingly.